charters/DataProcessingService/charter.md

```diff
@@ -0,0 +1,94 @@
+# [HCA Data Processing Pipelines](mailto:pipelines-team@data.humancellatlas.org)
+
+## Description
+
+The HCA Data Processing Pipelines are cloud-based and portable analysis pipelines that perform transformative analysis on scientific data in the HCA Data Coordination Platform.
+
+## Objectives
+
+The HCA Data Processing Pipelines team develops all pipelines required by the HCA project for all transformative analyses on scientific data types in the HCA Data Coordination Platform (DCP). Pipelines created to respond to the addition of data types and assays and improvements to pipelines will be added iteratively and in collaboration with the HCA Analysis Working Group (AWG) and the community. 
+
+## Definitions
+
+__Community pipeline:__ A pipeline that is not officially used as a standard in the HCA project, but runs in the DCP and is operated by the Data Processing Service to enable community members to work with a diversity of submitted data until they have a standardized pipeline. These pipelines are not as vetted as standardized pipelines.  
+__HCA Analysis Working Group (AWG):__ A group of domain specialists that advise on and monitor scientific choices made in the Human Cell Atlas Project.  
+__HCA DCP Governance Group:__ A working group of the Human Cell Atlas project that advises the DCP, assuring alignment of DCP development efforts with the scientific project it supports.  
+__Onboarding:__ The process of taking an existing pipeline and engineering it to work in the HCA DCP.  
+__Scoping:__ The process of investigating the available resources and knowledge in a community around a specific assay to better understand how much work it would take to make a pipeline to process data from that assay.  
+__Standardized Pipeline:__ An official pipeline of the HCA project.  
+__Transformative Analysis:__ The processing of data that changes data or creates new data. This would not include processes that solely subset or reorganize data.
+
+## In-scope
+
+### Pipeline Creation
+
+* Scoping or review of scoping for new pipelines  
+* Suggesting, but not selection of, new standardized pipelines for the HCA DCP  
+* Creating all standardized HCA pipelines and documenting them  
+* Reviewing, on-boarding, and potentially creating community pipelines  
+* Reviewing community tools for use in data processing pipelines  
+* Working with the community to provide feedback on metadata standards for raw data in the HCA DCP to provide the required information to enable processing of the raw data  
```

Comment:
Perhaps:

 _Working with the community to provide feedback on HCA DCP metadata standards to ensure that required information is available to enable processing of raw data._